### ISSUE TITLE
Typo: The initial doc's value is not capitalized to match the actual file name.

### ISSUE THREAD

**Issue by 0xb1dd1e** [PDFTron/webviewer-react-sample#6]
The demo does not work as-is on Linux because the file is not found (incorrect lowercase):

https://github.com/PDFTron/webviewer-react-sample/blob/9eacc7184df27770400a8385e2d92fd7affddc2e/src/App.js#L13

https://github.com/PDFTron/webviewer-react-sample/blob/master/public/files/PDFTRON_about.pdf

**Comment by CorreyL** [comment]
Thanks for the heads up @0xb1dd1e, I've been able to re-produce the issue in this environment:

```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Raspbian
Description:    Raspbian GNU/Linux 10 (buster)
Release:        10
Codename:       buster
```

### Before

![image](https://user-images.githubusercontent.com/16601729/88323506-30fad000-ccd7-11ea-949e-5942e4363a8b.png)

### After

![image](https://user-images.githubusercontent.com/16601729/88323434-14f72e80-ccd7-11ea-9595-ca4fe89193f3.png)

Setting up a PR to fix this now. Thanks again!

**Comment by CorreyL** [comment]
Addressed in #7
